vault/data_source_aws_access_credentials.go

```diff
@@ -155,9 +155,17 @@ func awsAccessCredentialsDataSourceRead(d *schema.ResourceData, meta interface{}
 	d.Set("lease_start_time", time.Now().Format(time.RFC3339))
 	d.Set("lease_renewable", secret.Renewable)
 
+	rootPath := backend + "/config/root"
+	regionData, err := client.Logical().Read(rootPath)
```

Comment:
Uhm, this line appears to be a breaking change if existing source has not declared a region...

Reply:
I can confirm that this is a breaking change; I'm already getting reports of 405 errors from teams who haven't pinned their provider versions.

Reply:
Thank you for reporting this; we're tracking it in https://github.com/terraform-providers/terraform-provider-vault/issues/833